catalyst/callbacks/scheduler.py

```diff
@@ -118,7 +120,10 @@ def _scheduler_step(
             lr = scheduler.optimizer.param_groups[0]["lr"]
         else:
             scheduler.step()
-            lr = scheduler.get_lr()[0]
+            if version.parse(torch.__version__) < version.parse("1.6.0"):
```

Comment:
could you please add this flag to the setting, so we don't need to recompute it at each call?
our settings file https://github.com/catalyst-team/catalyst/blob/master/catalyst/settings.py
something like `IS_TORCH_ABOVE_16v` would be good enough